daemon/src/wallet.rs

```diff
@@ -304,6 +338,10 @@ pub struct BuildPartyParams {
 /// Private message to trigger a sync.
 struct Sync;
 
+pub struct Recover {
```

Comment:
```suggestion
pub struct Reinitialize {
```

Recover is too usecase specific IMO. From the wallet perspective, we are just re-initializing with a different seed.